CPAC/registration/tests/test_fsl_registration_connector.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from CPAC.registration.registration import FSL_registration_connector
+
+
+class AttrDict(dict):
+    def __getattr__(self, item):
+        value = self[item]
+        if isinstance(value, dict):
+            return AttrDict(value)
+        return value
+
+
+@pytest.mark.parametrize("sink_native_transforms", [True, False])
+def test_fsl_registration_connector(sink_native_transforms):
+    wf_name = "test_fsl_registration_connector"
+    cfg = AttrDict(
```

Comment:
(if we import `Configuration` instead of creating `AttrDict`)

```suggestion
    cfg = Configuration(
```